README.md

```diff
@@ -0,0 +1,95 @@
+# IntelliChef
+
+## Table of Contents
+- [Overview](#overview)
+- [Features](#features)
+- [Technical Requirements](#technical-requirements)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [Testing](#testing)
+- [Support](#support)
+- [License](#license)
+
+## Overview
+IntelliChef is a desktop application designed to help young adults discover recipes based on available ingredients while considering nutritional goals. The application addresses the challenge of finding suitable recipes that not only utilize ingredients on hand but also align with specific dietary requirements - a common difficulty faced by individuals new to cooking.
+
+### Contributors
+- Naren
+- Harpuneet
+- Enza
+- Fatir
+
+## Features
+1. **Ingredient Input**
+    - Upload images of available ingredients
+    - Manual ingredient entry option
+
+2. **Dietary Preferences**
+    - Set nutritional goals
+    - Specify dietary restrictions
+
+3. **Recipe Discovery**
+    - Browse recipes filtered by available ingredients
+    - Recipes automatically aligned with nutritional preferences
+
+4. **Detailed Recipe View**
+    - Enhanced recipe visualization
+    - Direct links to cooking instructions
+
+## Technical Requirements
+- Operating System: Desktop platforms supporting Java
+- Java Version: OpenJDK 22.0.2 or later
+- Build System: Maven
+
+### Dependencies
+- org.json:json:20240303
+- com.google.cloud:google-cloud-vertexai
+- com.formdev:flatlaf:3.0
+- JUnit and Mockito (for development)
+
+## Installation
```

Comment:
They will also need to configure google development environment (prerequisites) from the API documentation.